airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/email_events.json

```diff
@@ -8,6 +8,12 @@
     "appName": {
       "type": ["null", "string"]
     },
+    "bcc": {
+      "type": ["null", "array"]
+    },
+    "cc": {
+      "type": ["null", "array"]
```

Comment:
can we specify array items types?

Reply:
checked if there are any values in this fields in records we receive from test account, these fields are empty right now, so I can't specify the exact type of their items